.pipelines/run-mirror.yml

```diff
@@ -0,0 +1,30 @@
+# Azure DevOps Pipeline deploying the azure hosted environment
+trigger: none
+pr: none
+
+parameters:
+- name: location
+- name: environment
+  values:
+  - RP-INT
+  - RP-Prod
+- name: DST_AUTH
+- name: SRC_AUTH_GENEVA
+- name: SRC_AUTH_QUAY
+
+jobs:
+  - template: ./templates/template-job-deploy-azure-env.yml
+    parameters:
+      environment: ${{ parameters.environment }}
+      location: ${{ parameters.location }}
+      configFileName: $(config-file-name)
+      azureDevOpsJSONSPN: $(aro-v4-ci-devops-spn)
+      vsoProjectID: $(vso-project-id)
+      vsoConfigPipelineID: $(vso-config-pipeline-id)
+      vsoDeployerPipelineID: $(vso-deployer-build-id)
```

Comment:
if you include this, you will deploy an entire RP environment.  You don't want to do that.  You do however need the 3rd step from that file, which fetches the aro tool.